memcached-operator/deploy/olm-catalog/memcached-operator/0.12.0/memcached-operator.v0.12.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,176 @@
+              metadata:
+                labels:
+                  name: memcached-operator
+              spec:
+                containers:
+                  - command:
+                      - memcached-operator
+                    env:
+                      - name: WATCH_NAMESPACE
+                        valueFrom:
+                          fieldRef:
+                            fieldPath: metadata.annotations['olm.targetNamespaces']
+                      - name: POD_NAME
+                        valueFrom:
+                          fieldRef:
+                            fieldPath: metadata.name
+                      - name: OPERATOR_NAME
+                        value: memcached-operator
+                    image: REPLACE_IMAGE
+                    imagePullPolicy: Always
+                    name: memcached-operator
+                    resources: {}
+                serviceAccountName: memcached-operator
+      permissions:
+        - rules:
+            - apiGroups:
+                - ""
+              resources:
+                - pods
+                - services
+                - services/finalizers
+                - endpoints
+                - persistentvolumeclaims
+                - events
+                - configmaps
+                - secrets
+              verbs:
+                - '*'
+            - apiGroups:
+                - apps
+              resources:
+                - deployments
+                - daemonsets
+                - replicasets
+                - statefulsets
+              verbs:
+                - '*'
+            - apiGroups:
+                - monitoring.coreos.com
+              resources:
+                - servicemonitors
+              verbs:
+                - get
+                - create
+            - apiGroups:
+                - apps
+              resourceNames:
+                - memcached-operator
+              resources:
+                - deployments/finalizers
+              verbs:
+                - update
+            - apiGroups:
+                - ""
+              resources:
+                - pods
+              verbs:
+                - get
+            - apiGroups:
+                - apps
+              resources:
+                - replicasets
+              verbs:
+                - get
+            - apiGroups:
+                - cache.example.com
+              resources:
+                - '*'
+              verbs:
+                - '*'
+          serviceAccountName: memcached-operator
+    strategy: deployment
+  installModes:
+    - supported: true
+      type: OwnNamespace
+    - supported: true
+      type: SingleNamespace
+    - supported: false
+      type: MultiNamespace
+    - supported: true
+      type: AllNamespaces
+  keywords:
+    - memcached
+    - app
+  maintainers:
+    - email: corp@example.com
+      name: Some Corp
+  maturity: alpha
+  provider:
+    name: Example
+    url: www.example.com
+  replaces: memcached-operator.v0.11.0
```

Comment:
`spec.replaces` should be removed as per #89.

Reply:
I think we should keep it as it has been doing since the go is just upgrade it. Then, also I think that we should discuss this approach of tags and etc..  Or at least do another pr to change it in order to make clear the purpose of this change. could we agree with?

Reply:
As I stated before (linked in #89) having `spec.replaces` present when it’s value is not in the package manifest will prevent thIs CSV from installing. Remove this line in this PR and we can discuss how we want to deal with CSV versions and package manifests in #89.

Note that all other CSV versions are broken because their `spec.replaces` reference CSVs not in their package manifests, so following prior work is not the right solution here.

Reply:
@camilamacedo86 based on this discussion, looks like the replaces should be removed in order to allow this PR to move forward.

Reply:
Hi @jmrodri, 

@estroz is suggesting we change the template and do as [here](https://github.com/operator-framework/operator-sdk-samples/issues/89#issuecomment-552048899). However, it is not part of the goal to upgrade the file and it was a big discussion which in my understanding do not achieve a result yet in the previous upgrade to 0.11.   

IHMO: we should not change how things are and bring it to eng meeting. I do not think that we should create a new tag of CSV for each release for example. So I think we can leave it to be done in another PR if it is the case. 

@estroz do you agree? Could we leave the CSV as it is now and then creates a PR to change it and/or how it has been working?